### PR TITLE
[AUTH-546] IDS 1.8.0.1

### PR DESCRIPTION
**Pull request by tiagosalvado10**

### CODE REVIEW

File: helm/alfresco-identity-service/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: alfresco-identity-service
-version: 7.0.0
-appVersion: 1.8.0
+version: 8.0.0
```
Comment (gionn) [inline helm/alfresco-identity-service/Chart.yaml:3]: I would not bump the chart major version if we are releasing just an hotfix, `7.0.1` should be fine

File: README.md [hunk 2]
```diff
@@ -267,7 +267,7 @@ helm install $RELEASENAME alfresco-stable/alfresco-identity-service --devel \
   --namespace $DESIREDNAMESPACE
 ```
 
-For further details see [Setting a Custom Realm](https://github.com/codecentric/helm-charts/tree/master/charts/keycloak#setting-a-custom-realm).
+For further details see [Setting a Custom Realm](s).
```
Comment (gionn) [inline README.md:270]: typo?

**Review by gionn — CHANGES_REQUESTED** [review]

File: helm/alfresco-identity-service/Chart.yaml (hunk 1 above)
Comment (tiagosalvado10) [reply helm/alfresco-identity-service/Chart.yaml]: Changed version from `8.0.0` to `7.0.1`

File: README.md (hunk 2 above)
Comment (tiagosalvado10) [reply README.md]: Fixed link.

**Pull request merged**